src/Altinn.AccessManagement.Core/Helpers/RightsHelper.cs

```diff
@@ -67,6 +68,16 @@ public static RightsQuery GetRightsQuery(int userId, int fromPartyId, string res
                 };
             }
 
+            if (!string.IsNullOrEmpty(serviceCode) && !string.IsNullOrEmpty(serviceEditionCode))
```

Comment:
The added ServiceCode/ServiceEditionCode parameters and RightsQuery return can be removed as this is never used since Altinn 2 SBL Bridge handles the actual request.

Reply:
Removed these unused parameters